GeneralisedFilters/src/algorithms/particles.jl

```diff
@@ -250,10 +239,8 @@ function propogate(
         ref_state
     end
 
-    # TODO: make this type consistent
-    # Will have to do a lazy zero or change propogate to accept a particle (in which case
-    # we'll need to construct a particle in the RBPF predict method)
-    return new_x, 0.0
+    # TODO: replace this with nothing (unweighted particle)
+    return new_x, 0
```

Comment:
I guess I missed one
```suggestion
    return new_x, false
```